integration-tests/src/test/java/oracle/weblogic/kubernetes/ItLBNginx.java

```diff
@@ -0,0 +1,456 @@
+  private static HelmParams nginxHelmParams = null;
+  private static Path tlsCertFile;
+  private static Path tlsKeyFile;
+  private static LoggingFacade logger = null;
+
+  // domain constants
+  private static final int replicaCount = 2;
+  private static final int MANAGED_SERVER_PORT = 7100;
+  private static final int ADMIN_SERVER_PORT = 7001;
+  private static final String clusterName = "cluster-1";
+
+  /**
+   * Assigns unique namespaces for operator and domains.
+   * Pull WebLogic image if running tests in Kind cluster.
+   * Installs operator and create domain.
+   *
+   * @param namespaces injected by JUnit
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(3) List<String> namespaces) {
+    logger = getLogger();
+    logger.info("Assign a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace is null");
+    String opNamespace = namespaces.get(0);
+
+    // get unique domain namespaces
+    logger.info("Get unique namespaces for WebLogic domain1 and domain2");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+
+    // get a unique Nginx namespace
+    logger.info("Assign a unique namespace for Nginx");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    nginxNamespace = namespaces.get(2);
+
+    // set the service account name for the operator
+    String opServiceAccount = opNamespace + "-sa";
+
+    // install and verify operator with REST API
+    installAndVerifyOperator(opNamespace, opServiceAccount, true, 0, domainNamespace);
+
+    // This test uses the operator restAPI to scale the domain. To do this in OKD cluster,
+    // we need to expose the external service as route and set tls termination to  passthrough
+    logger.info("Create a route for the operator external service - only for OKD");
+    createRouteForOKD("external-weblogic-operator-svc", opNamespace);
+    // Patch the route just created to set tls termination to passthrough
+    setTlsTerminationForRoute("external-weblogic-operator-svc", opNamespace);
+
+    // create pull secrets for WebLogic image when running in non Kind Kubernetes cluster
+    // this secret is used only for non-kind cluster
+    createSecretForBaseImages(domainNamespace);
+
+    for (int i = 1; i <= numberOfDomains; i++) {
+      domainUids.add("wls-domain" + i);
+    }
+
+    createMultipleDomainsSharingPVUsingWlstAndVerify(
+        domainNamespace, wlSecretName, ItLBNginx.class.getSimpleName(), numberOfDomains, domainUids,
+        replicaCount, clusterName, ADMIN_SERVER_PORT, MANAGED_SERVER_PORT);
+
+    // build and deploy app to be used by all test cases
+    buildAndDeployClusterviewApp(domainNamespace, domainUids);
+
+    // install Nginx ingress controller for all test cases using Nginx
+    installIngressController();
+  }
+
+  /**
+   * Verify WebLogic admin console is accessible through NGINX path routing with HTTPS protocol.
```

Comment:
Modify the description:  Verify the WebLogic Administration Console form both domain is accessible thru a path routing based single NGIX  LoadBalancer  using HTTP protocol.

Reply:
fixed